lib/parse.js

```diff
@@ -72,6 +75,22 @@ module.exports = function (input) {
             pos = next + 1;
             code = value.charCodeAt(pos);
 
+        //Comments
+        } else if (code === slash && value.charCodeAt(pos + 1) === star) {
+            token = { type: 'comment', sourceIndex: pos};
+
+            next = value.indexOf(commentEnd, pos);
+            if (next < 0) {
+                token.unclosed = true;
+                next = value.length;                
+            }
+
+            token.value = value.slice(pos + commentStart.length, next);
```

Comment:
We know that `commentStart.length` is two. It's intuitive constant. Better use comment instead of variable.